electron-builder.json

```diff
@@ -50,17 +50,14 @@
   "mac": {
     "category": "public.app-category.productivity",
     "target": [
-      "tar.gz"
-    ],
-    "extraResources": [
-      {
-        "filter": [
-          "LICENSE.txt",
-          "NOTICE.txt"
-        ]
-      }
+      "dmg"
     ]
   },
+  "dmg": {
+    "background": "resources/osx/DMG_BG.png",
+    "iconSize": 140,
```

Comment:
Do we need to specify size here?

Reply:
https://github.com/mattermost/mattermost-webapp/pull/208#issuecomment-399727075

Note: We might make change when creating our own background.